rpi-setup.sh

```diff
@@ -129,24 +152,28 @@ StandardOutput=syslog
 StandardError=syslog
 WorkingDirectory=$dir
 User=$SUDO_USER
+Environment=EXTERNALPORT=$primaryPort
 [Install]
 WantedBy=multi-user.target
 systemdfile
 
 sudo chmod 755 $systemd
 
+if [ $secondaryPort != "" ]; then
+  secondListen="ListenStream=$secondaryPort"
+fi
+
 cat > $socket <<socket
 [Socket]
-ListenStream=$port
+ListenStream=$primaryPort
+$secondListen
```

Comment:
Don’t you need to repeat `ListenStream` on the second line?

Reply:
See above $secondListen includes that. This to handle the case where they choose not to use ssl and there will be only one ListenStream